app/components/public/speaker-item.js

```diff
@@ -1,7 +1,26 @@
 import Ember from 'ember';
 
-const { Component } = Ember;
+const { Component, computed } = Ember;
 
 export default Component.extend({
-  classNames: ['column', 'four', 'wide']
+  classNameBindings: ['style'],
+
+  style: computed(function() {
+    const isFeatured = this.get('isFeatured');
+    if (isFeatured) {
+      return 'column eight wide';
+    } else {
+      return 'column four wide';
+    }
+  }),
+  didInsertElement() {
+    const isFeatured = this.get('isFeatured');
+    if (isFeatured) {
+      this.$('.speaker-item').addClass(' ui grid stackable');
```

Comment:
Make these class bindings via the template itself so that even if `isFeatured` changes in real-time, we can see class name changes. (Not useful right now. But just making it future proof)